astropy/modeling/fitting.py

```diff
@@ -1657,7 +1657,20 @@ class LMLSQFitter(_NLLSQFitter):
     def __init__(self, calc_uncertainties=False):
         super().__init__("lm", calc_uncertainties, True)
 
-    def __call__(self, model, *args, **kwargs):
+    @fitter_unit_support
+    def __call__(
+        self,
+        model,
+        x,
+        y,
+        z=None,
+        weights=None,
+        maxiter=DEFAULT_MAXITER,
+        acc=DEFAULT_ACC,
+        epsilon=DEFAULT_EPS,
+        estimate_jacobian=False,
+        filter_non_finite=False,
+    ):
```

Comment:
I suspect this had not been rebased to #17033 and thus the explicit signature is now missing the `inplace` kwarg that was added there.